public/assets/mapIcons/empty-icon.svg

```diff
@@ -0,0 +1,2 @@
+<svg width="1" height="1" xmlns="http://www.w3.org/2000/svg">
```

Comment:
@jimboweb curious what the rationale is for using an empty SVG for the marker icon? Why not use Leaflet's default marker?

Reply:
changed to default marker